frontend/src/modules/search/components/SearchBox.js

```diff
@@ -176,6 +224,7 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
         const statuses = this.getInitialStatuses();
         const extras = this.getInitialExtras();
         const tags = this.getInitialTags();
+        let timeRange = this.getInitialTimeRange();
```

Comment:
I like consistency too, but in this case I really think this should be just `let timeRange = null;` because I believe simplicity > consistency. :-)

Reply:
Good point. I'll remove `getInitialTimeRange()`.